src/main/graphql/GetParentFolder.graphql

```diff
@@ -0,0 +1,6 @@
+query getParentFolder($consignmentId: UUID!) {
```

Comment:
Maybe this could be something more general like `getConsignmentFolderDetails` since it also returns the number of files in the folder?

Reply:
Ah good idea! :+1: